blockchain/types/accountkey/account_key.go

```diff
@@ -108,11 +108,12 @@ func NewAccountKey(t AccountKeyType) (AccountKey, error) {
 	return nil, errUndefinedAccountKeyType
 }
 
-func ValidateAccountKey(from common.Address, accKey AccountKey, recoveredKeys []*ecdsa.PublicKey, roleType RoleType) error {
-	if !accKey.Validate(roleType, recoveredKeys, from) {
-		return errInvalidSignature
+func ValidateAccountKey(from common.Address, accKey AccountKey, recoveredKeys []*ecdsa.PublicKey, roleType RoleType, currentBlockNumber uint64) (error, int) {
```

Comment:
When a function returns both value and error, placing error last is a more common pattern in Klaytn code.

Reply:
It has been reverted, thanks!